test/data/super.layout

```diff
@@ -25,6 +25,7 @@
   "steps": [
    {
     "_type": "step",
+    "certConstraints": [],
```

Comment:
Yeah, should probably rename this `cert_constraints` to be consistent with all of these other properties.